<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AVSGenerator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Seleção</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vídeo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Legenda:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="caminhoVideo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="caminhoLegenda">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="selecaoVideo">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="selecaoLegenda">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>109</y>
      <width>371</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Opções</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Início:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Fim:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de vídeo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de legenda:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Regras adicionais:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="corte">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Corte</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="regras">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Regras</string>
     </property>
    </widget>
    <widget class="QComboBox" name="tipoVideo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>FFDShow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FFMS2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="tipoLegenda">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>VSFilter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VSFilterMod</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XY-VSFilter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LibAss</string>
      </property>
     </item>
    </widget>
    <widget class="QTextEdit" name="regrasTexto">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="corteInicio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="corteFim">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>371</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gerar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gerar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Saída:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="caminhoSaida">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="selecaoSaida">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIn_cio">
    <property name="title">
     <string>Início</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionSalvar_Como"/>
   </widget>
   <widget class="QMenu" name="menuOpcoes">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="actionConfigurar"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Informações</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="actionAutor"/>
    <addaction name="actionURL"/>
   </widget>
   <addaction name="menuIn_cio"/>
   <addaction name="menuOpcoes"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionSalvar_Como">
   <property name="text">
    <string>Salvar Como</string>
   </property>
  </action>
  <action name="actionConfigurar">
   <property name="text">
    <string>Configurar</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionAutor">
   <property name="text">
    <string>Autor</string>
   </property>
  </action>
  <action name="actionURL">
   <property name="text">
    <string>URL</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
